app/models/post.rb

```diff
@@ -0,0 +1,17 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :likes
+  has_many :comments
+
+  after_save :update_posts_counter
+
+  def update_posts_counter
+    author.increment!(:posts_counter)
+  end
+
+  def five_most_recent_comments
+    post.comments.order(updated_at: :desc).first(5)
+  end
+
+  private :update_posts_counter
```

Comment:
- [ ] [OPTIONAL] Great implementaton @fernando-silvabr66 , however, I would like ti to suggest that you can use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the #update_post_counter method every time a new post has been saved to the database, then make the update_post_counter method private.

Reply:
Thanks for the suggestion. I will study it a bit and try to implement in the next project's phase.